### PR TITLE
Feature/rz about

### PR DESCRIPTION
**Pull request by nbonatsakis**

### CODE REVIEW

**Comment by nbonatsakis** [comment]
@mr-fixit @KingOfBrian @arrouse Can one or more of you guys have a look at this? I think there's more we can add to this, namely Eric's acknowledgements stuff, but it's a start.

File: RZUtils.podspec [hunk 1]
```diff
@@ -7,191 +7,197 @@ Pod::Spec.new do |s|
...
   #
-  
+
   s.subspec "Components" do |ss|
-    
+
+    ss.subspec "RZAbout" do |sss|
+      sss.source_files = "RZUtils/Components/RZAbout/*.{h,m}"
+      sss.resources = "RZUtils/Resources/RZAbout/*.png"
+      #sss.resource_bundle = { 'RZAbout' => 'RZUtils/Resources/RZAbout/*.png"' }
```
Comment (mr-fixit) [inline RZUtils.podspec]: #req if you don't need this line, why not delete?

File: RZUtils/Components/RZAbout/RZAbout.m [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+// LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+// WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+
+@import UIKit;
+#import "RZAbout.h"
+#import <sys/utsname.h> // import it in your header or implementation file.
```
Comment (mr-fixit) [inline RZUtils/Components/RZAbout/RZAbout.m]: #?  a comment for an import?   

I guess deviceName() was lifted straight from http://stackoverflow.com/questions/11197509/ios-how-to-get-device-make-and-model

File: RZUtils/Components/RZAbout/RZAbout.m [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+    [logoView addSubview:buildLabel];
+
+    return logoView;
+}
+
+#pragma mark - App Info
+
++ (NSString *)systemVersion
+{
+    return [UIDevice currentDevice].systemVersion ?: @"Unknown";
```
Comment (mr-fixit) [inline RZUtils/Components/RZAbout/RZAbout.m]: #? why use hardcoded English `'Unknown'` here (and next 3 methods) when you use `NSLocalizedString()` macro in `appInfoText()`?

**Comment by mr-fixit** [comment]
@nbonatsakis looks fine, except for the minor issues and questions above. I bet I'm not the first to wish RZUtils had a sample app.

File: RZUtils/Components/RZAbout/RZAbout.m (hunk 2 above)
Comment (nbonatsakis) [inline RZUtils/Components/RZAbout/RZAbout.m]: It sure was! Will fix that.

File: RZUtils/Components/RZAbout/RZAbout.m (hunk 3 above)
Comment (nbonatsakis) [inline RZUtils/Components/RZAbout/RZAbout.m]: It's an edge case that shouldn't happen, but I'll change that.

**Comment by nbonatsakis** [comment]
@mr-fixit Fixed those issues and added some very basic smoke tests, mostly to make sure things compile and can do the baseline.
